frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -384,121 +146,6 @@ const plugin: Plugin<ConsumedExtensions> = [
       disallowed: [OCS_FLAG, ODF_MODEL_FLAG],
     },
   },
```

Comment:
are these Routes still used somewhere ? I can see we did not remove them...

Reply:
This route used in some places like ocs-system-dashboard.tsx and few cypress testings.

Reply:
is it still used odf-console ?? just to point out u r making these changes for OCP/ODF 4.13 version

Reply:
Not used in odf-console or ODF.